cepy/__init__.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2017 Gamalon, Inc.
```

Comment:
The Apache software foundation states that you should attach a specific header to each file that is part of the licensed software: https://www.apache.org/licenses/LICENSE-2.0#apply. In addition, it would be nice to have the first line of the header look like
```python
# nameoffile.py - one line describing purpose of file
```

Reply:
Done